source/extensionPoints/util.py

```diff
@@ -75,12 +90,16 @@ def __init__(self):
 		#: Registered handler functions.
 		#: This is an OrderedDict where the keys are unique identifiers (as returned by _getHandlerKey)
 		#: and the values are weak references.
-		self._handlers = collections.OrderedDict()
+		self._handlers = OrderedDict[
+			HandlerKeyT,
+			Union[BoundMethodWeakref[HandlerT], AnnotatableWeakref[HandlerT]]

```

Comment:
this type also might be worth turning into a named variable, as it is used repeatedly

Reply:
Same issue as abbove, nesting TypeVars in variables doesn't seem to be respected by PyRight.